packages/typo3-docs-theme/resources/template/body/version-change.html.twig

```diff
@@ -1,16 +1,15 @@
-<div class="versionchange {{ node.type }}">
-    <p class="versionmodified">
+<section class="versionchange {{ node.type }}">
```

Comment:
should we use "section" here? a section should usually have a heading and for now we only use it for sections in the sense of rst-section so containing one headline and the content.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section?retiredLocale=de "Sections should always have a heading"

Reply:
Yup, we have a heading (`<header>`). Another solution would be just use a `<div>` and remove the `<header>`.

Reply:
The header is not a heading. A heading is <h1> - <h6>

Reply:
`h1`- `h6`